### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by dxni**
I fixed the observations :)

### CODE REVIEW

File: Homework2/MismatchExercise.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        
+        int resultSecondTest = mismatch(secondTestArrayOne, secondTestArrayTwo);
+        System.out.println(resultSecondTest); //result should be 4
+
+
+    }
+
+    public static int mismatch(int[] left, int[] right) {
+        //Fill your code here
+        return -1;
```
Comment (uabarahona) [inline Homework2/MismatchExercise.java]: You need to finish this :)

**Review by uabarahona — DISMISSED** [review]
Good solutions and smart use of Arrays.Sort, I have some suggestions, please check it :)

File: Homework2/MismatchExercise.java (hunk 1 above)
Comment (dxni) [reply Homework2/MismatchExercise.java]: soon fixed

**Comment by dxni** [comment]
I updated the fix for homework 2 and homework 3

**Pull request merged**